app/src/main/kotlin/eu/quiqua/gloomhaven/statstracker/features/stats/viewModel/StatsViewModel.kt

```diff
@@ -17,7 +20,43 @@ class StatsViewModel @Inject constructor(application: android.app.Application) :
     }
 
     @Inject
-    lateinit var model: String
+    lateinit var stats: Stats
+
+    val mutableHp: MutableLiveData<Int> = MutableLiveData()
+    val mutableXp: MutableLiveData<Int> = MutableLiveData()
```

Comment:
this livedata thing is confusing.

it would be better if my model itself could be observed somehow oO